ecosystem/typescript/sdk/src/providers/indexer.ts

```diff
@@ -92,7 +93,9 @@ export class IndexerClient {
    * @param graphqlQuery A GraphQL query to pass in the `data` axios call.
    */
   async queryIndexer<T>(graphqlQuery: GraphqlQuery): Promise<T> {
-    const { data } = await axios.post(this.endpoint, graphqlQuery);
+    const { data } = await axios.post(this.endpoint, graphqlQuery, {
+      headers: { "x-aptos-client": `aptos-ts-sdk/${VERSION}` },
```

Comment:
We're using this header key in two places now, let's make it a constant.

Similarly we're building the value of the header in two places, let's make it a function.

Reply:
imo we can just make the header a global constant instead of function. .e.g. something like `export const commonHeaders = { .... }` and then just pass it to the client calls